src/components/grid/grid.xml

```diff
@@ -25,6 +24,7 @@
         onComposerContentFocused="props.onComposerContentFocused"
         onComposerCellFocused="props.onGridComposerCellFocused"
         focus="props.focusComposer"
+        t-on-contextmenu="onInputContextMenu"
```

Comment:
Yeah this won't work. You're trying to pass a props in a component with a `t-on-*` ... The props onInputContextMenu is always undefined then. 

Maybe write some tests ^^' 

Also is this part really necessary ? The task only mentions click position. I don't have a contextMenu button on my keyboard to check if it works or not in 16.0 and master.

Reply:
> Yeah this won't work. You're trying to pass a props in a component with a `t-on-*` ... The props onInputContextMenu is always undefined then.

Oops, that was the mistake. I copied a line from above and forgot to change the name of the props 😅
But it's still working? Weird.
 
> Also is this part really necessary ? The task only mentions click position. I don't have a contextMenu button on my keyboard to check if it works or not in 16.0 and master.

Shift+F10 ?